pxr/usdImaging/lib/usdviewq/appController.py

```diff
@@ -1767,6 +1775,7 @@ def _playClicked(self):
             self._primViewUpdateTimer.stop()
             self._playbackIndex = 0
         else:
+            self._ui.frameSlider.setTracking(False)
```

Comment:
I imagine this will cause an issue when '_Redraw On Frame Scrub_' is checked. Clicking play, then stop, will result in tracking being disabled where instead it should be reverting back to the value stored in the UI.

`
self._ui.frameSlider.setTracking(self._ui.redrawOnScrub.isChecked())
`